### ISSUE TITLE
Change items around from various shops to enhance RP.

### ISSUE THREAD

**Issue by Amoxicot** [InZidiuZ/op-framework-issue-tracker#2015]
Assurance:
Yes

Summary:
Changing the items available from the convenience stores to aid Burger Shot and Bean Machine and taking the Burger Show marker from the Arena.

Reason:
Removing certain items from the convenience stores would enable items to be moved to JUST Bean Machine or Burger Shot and in turn would move more business towards those two places and create more RP for them. As well as removing the Burger Shot store from the Arena so people can't just go there to avoid the RP.

EXAMPLE:
![Items](https://user-images.githubusercontent.com/91511732/135010148-5fd248c1-73f6-4869-b9d0-ac731315752c.png)
------------------------------------------------------------------------------------------------------------------------------------------
The crossed out items designate removing from that specific store and the **BOLD** lettering means that item is ADDED to that specific store.

**Comment by xEleyna** [comment]
Sounds good, especially since customers keep complaining why burger shot doesn’t sell fries 😆

**Comment by Amoxicot** [comment]
I am actually a little surprised that this isn't getting any sort of traction. This is something that would most definitely aid in RP.

**Comment by PedroTheTurtle** [comment]
This would definitely be a great change to the city. Making it so people have more of a reason to go to places like Bean Machine and Burger Shot for food instead of just going to the convenience stores. I agree that it definitely will add to more rp in the city.

**Comment by nimsaJChu** [comment]
i agree, that would improve the rp a lot. although i gotta say its a self driving chip, not a self destruct 🤣

**Comment by ntashaa** [comment]
i agree, scrap the apple though we're promoting obesity only
